linux/coalescer.go

```diff
@@ -58,6 +58,13 @@ func (c *Coalescer) Coalesce(ctx context.Context, layerArtifacts []*indexer.Laye
 		for db, pkgs := range tmp {
 			dbs[db] = pkgs
 		}
+
+		if len(artifacts.Files) > 0 {
+			c.ir.Files = make(map[string][]claircore.File)
+		}
+		for _, f := range artifacts.Files {
+			c.ir.Files[artifacts.Hash.String()] = append(c.ir.Files[artifacts.Hash.String()], f)
```

Comment:
nit: may be nice to set `artifacts.Hash.String()` to a variable, so we don't call it so many times